app/workers/last_articles_per_user.rb

```diff
@@ -0,0 +1,6 @@
+class LastArticlesPerUser
+  include Sidekiq::Worker
+  def perform()
+    BlogMailer.send_last_articles_per_user().deliver_now!
```

Comment:
Its okay to make this mailer like this. Because if you use sidekiq queues can get bigger and bigger